### PR TITLE
OSSRH-82400 Publish to Sonatype Open Source Project Repository

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.sonatype.org/browse/OSSRH-82400

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2134,12 +2122,29 @@
...
+               <executions>
+                  <execution>
+                     <id>sign-artifacts</id>
+                     <phase>verify</phase>
+                     <goals>
+                        <goal>sign</goal>
+                     </goals>
+                     <configuration>
+                        <keyname>9E31AB27445478DB</keyname>
+                        <passphraseServerId>9E31AB27445478DB</passphraseServerId>
```
Comment (pruivo) [inline pom.xml:2143]: shouldn't these be properties and be set in Jenkins `settings.xml`?

Comment (tristantarrant) [reply pom.xml:2143]: The key name is public: https://keyserver.ubuntu.com/pks/lookup?search=9E31AB27445478DB&fingerprint=on&op=index

Comment (pruivo) [reply pom.xml:2143]: but if you want to run/sign locally, you need to change the hardcoded values :)

Comment (tristantarrant) [reply pom.xml:2143]: I don't understand the need to sign artifacts with the non-official key

Comment (pruivo) [reply pom.xml:2143]: fair enough. :+1:

**Comment by pruivo** [comment]
@tristantarrant deploying locally, the modules inside the `build` folder are not being signed.

**Comment by tristantarrant** [comment]
I've moved gpg and nexus staging plugin config and profiles over to `bom/configuration/pom.xml` so that they apply to every module now

**Review by pruivo — APPROVED** [review]
LGTM! It is working locally.

**Comment by pruivo** [comment]
replaced by https://github.com/infinispan/infinispan/pull/10225

**Pull request closed without merging**